clm_demeter/clm_transition_priorities.py

```diff
@@ -0,0 +1,120 @@
+import pandas as pd
+import numpy as np
+
+__author__ = 'Min Chen and Eva Sinha'
+__email__  = 'min.chen@pnnl.gov and eva.sinha@pnnl.gov'
+__copyright__ = 'Copyright (c) 2017, Battelle Memorial Institute'
+
+class CLMTransitionPriorities:
+    """Develop transition priorities for CLM types
```

Comment:
It would be better to describe what each factor represents for in the comments.
e.g., geographical factor: Boreal, Temperate, tropical etc
leaf shape: broadleaf, needleleaf etc

Reply:
Thats a good point @chenminbnu. I will add more comments describing the factors once I receive comments from @crvernon also.